schemas/lioncore.serialization.schema.json

```diff
@@ -0,0 +1,520 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "LIonCore-serialization",
```

Comment:
I would call it `LionCore-serialization`

Reply:
See above - can always change it later.